dev/devicelab/lib/tasks/perf_tests.dart

```diff
@@ -402,7 +402,15 @@ class CompileTest {
         watch.start();
         await flutter('build', options: options);
         watch.stop();
-        final String appPath =  '$cwd/build/ios/Release-iphoneos/Runner.app/';
+        final Directory appBuildDirectory = dir(path.join(cwd, 'build/ios/Release-iphoneos'));
```

Comment:
@christopherfujino here we go again.
https://github.com/flutter/flutter/pull/46928 was still hard-coding `.app`.  We don't know what it will be called between different perf tests, so find it dynamically.